### PR TITLE
French Translation

### PR DESCRIPTION
**Pull request by PoissonBallon**
I made a new PR with french translation, sorry for the duplicate (https://github.com/vapor/website/pull/34)

### CODE REVIEW

File: src/i18n/fr.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+      start: 'Premier pas',
+      join: 'Discuter'
+  },
+  nav: {
+      code: 'Code',
+      cloud: 'Cloud',
+      docs: 'Docs'
+  },
+  bio: {
+      first: '<a target="_blank" href="https://medium.com/@codevapor/vapor-3-0-0-released-8356fa619a5d">Ultra-rapide</a>, Asynchrone, architecture "event-driven" et propulsé par <a target="_blank" href="https://github.com/apple/swift-nio">SwiftNIO by Apple</a>.',
```
Comment (tanner0101) [inline src/i18n/fr.js]: Does `Asynchrone` need to be capitalized?

Comment (PoissonBallon) [reply src/i18n/fr.js]: Nop

**Review by tanner0101 — APPROVED** [review]
Merci beaucoup !

**Comment by penny-coin** [comment]
Hey @PoissonBallon, you just merged a pull request, have a coin! 

You now have 1 coins.

**Comment by tanner0101** [comment]
screen shot 2019-02-27 at 4 41 41 pm

**Comment by PopFlamingo** [comment]
Wow ! Merci @PoissonBallon !

**Pull request merged**